src/main/java/com/stackify/api/common/log/LogAppender.java

```diff
@@ -127,6 +127,16 @@ public void append(final T event) {
 			return;
 		}
 
+		// skip internal logging
+		
+		String className = eventAdapter.getClassName(event);
+		
+		if (className != null) {
+			if (className.startsWith("com.stackify.api.")) {
```

Comment:
Just a comment on this one, so no suggested changes. I'm a huge fan of using static constants or enumerators for string literals. This isn't a major deal, but might be useful to put the Stackify package in a constant somewhere in case we need to use it in multiple places. It could be a situation where the class package isn't used very often, so this is probably a non-issue.